app/models/video.rb

```diff
@@ -0,0 +1,27 @@
+class Video < ActiveRecord::Base
+  validate :verify_youtube_video
+
+  validates :slot, numericality: { only_integer: true,
+                                   greater_than_or_equal_to: 1,
+                                   less_than_or_equal_to: 4
+                                  },
+                                  allow_blank: true
+
+  validates_inclusion_of :page, in: ["tools and resources",
```

Comment:
There are many videos on different pages and so I think when you create a video, you should be able to designate what page it's on?